README.md

```diff
@@ -45,6 +44,12 @@ To select a specific part of the screen you can either use `-g <geometry>`, or u
 wf-recorder -g "$(slurp)"
 ```
 
+You can record screen and sound at the same with
+
+```
+wf-recorder --audio --file=$HOME/recording_$(date "+%F_%H-%M-%S").mp4
```

Comment:
I think this is a good idea but we should keep examples as simple as possible, so please drop the fancy file name. Maybe just `with_audio.mp4`

Reply:
I admit the file name looks fancy^^. But it's also very useful to have the current date and time in it: no more overwriting of recordings. I also do this for my screenshots. Do you want to reconsider and leave the fancy name or do you want to have a static file name `with_audio.mp4`?

Reply:
I admit a fancier name is definitely useful, but it is beside the point when showing an example.